src/get-coverage.ts

```diff
@@ -28,8 +32,10 @@ const getCoverage = async (path: string): Promise<CommentData> => {
     commentData.branches.percent = (commentData.branches.hit / commentData.branches.found) * 100;
 
     return commentData;
-  } catch (err) {
-    throw new Error(err);
+  } catch (error) {
+    throwError(error, 'Failted to parse from ./coverage/lcov.parse.');
```

Comment:
This path is customizable so we should display the value of that setting and not a hardcoded string